src/otx/algo/callbacks/adaptive_train_scheduling.py

```diff
@@ -62,6 +62,16 @@ def on_train_start(self, trainer: Trainer, pl_module: LightningModule) -> None:
             trainer.check_val_every_n_epoch = adaptive_check_val_every_n_epoch
             self._change_early_stopping_patience(trainer.callbacks, adaptive_check_val_every_n_epoch)
             self._change_lr_scheduler_frequency(trainer.lr_scheduler_configs, adaptive_check_val_every_n_epoch)
+        else:
+            # To get the consistent result with the OTX1.x, the patience should be changed
+            # Since OTX1.x scheduler works with >= patience, however, OTX2.x scheduler works with > patience
+            for config in trainer.lr_scheduler_configs:
+                if hasattr(config, "scheduler") and hasattr(config.scheduler, "patience"):
+                    msg = (
+                        "The LR scheduler for OTX2 operates when the patience exceeds the set value, "
+                        "hence, the configured value is reduced by 1."
+                    )
+                    config.scheduler.patience -= 1
```

Comment:
What is the minimum value for `config.scheduler.patience`? 0 or 1? Please clip it with the minimum.

Reply:
Rather than adding this code, is it clear to change the default configurations (`src/otx/recipe`)?

Reply:
I also think that way, however, in that case, logic could be different.
If I get your point, you might mean that changing the value of `config.scheduler.patience=5 --> 4` 

When we consider the `adaptive_interval=2, original_patience=6` at the OTX1.x, 
* OTX1.x behavior : patience = (6/2) = 3 with reduce at  >= patience
* OTX2.x behavior (this PR): (6/2) -1 = 2 with reduce at > patience
* OTX2.x behavior (config change, -1 at the config level): (5/2) -1 = 1 with reduce at > patience

Reply:
Fixed the logic as we discussed, https://github.com/openvinotoolkit/training_extensions/pull/3060/commits/827720616db9ddb9b367f574254464abe34eb468